examples/infinite-scrolling/react-window-infinite-loader.tsx

```diff
@@ -0,0 +1,58 @@
+// Run `yarn webpack-dev-server` then open the http://localhost:8080/demo1 (or other port in the cli)
+import React, { Fragment, useCallback } from 'react'
+import { FixedSizeList as List } from 'react-window'
+import InfiniteLoader from 'react-window-infinite-loader'
+import ReactDOM from 'react-dom'
+import { useSWRInfinite } from '../../src/use-swr-infinite'
+import { load } from './data'
+import { ListChildComponentProps } from 'react-window'
+
+function Row(props: ListChildComponentProps) {
+  const { style, data } = props
+  return (
+    <div className="ListItem" style={style}>
+      Item {data?.id} Data: {data?.data}
+    </div>
+  )
+}
+
+export default function App() {
+  const {isValidating, data, error, page, setPage} = useSWRInfinite((index, previousPageData) => {
+    if (previousPageData && previousPageData.length === 0) return null
+   return previousPageData.last()?.id
```

Comment:
When `index` is 0, which means that's the first page, you should return `0` here because there's no previous page data.

Reply:
Umm I learn it from your PR

Reply:
Check the second example in #435:

```js
  (index, previousPageData) => {
    // reached the end
    if (previousPageData && previousPageData.data.length === 0) return null

    // first page, previousPageData is null
    if (index === 0) return `/users?limit=10`

    return `/users?cursor=${previousPageData.nextCursor}&limit=10`
  },
```

Reply:
oh, I find it, cause it seems like the key `0` is considered as falsy. I suggest using `=== null` check

Reply:
Yeah and you should return a string :)